config.yml

```
@@ -94,39 +94,17 @@ bulk_data_jwks:
     
 # preset fhir servers: optional. Minimally requires name, uri, module, optional inferno_uri, client_id, client_secret, scopes, instructions link
 presets:
-  reference_server:
-    name: FHIR Reference Server
-    uri: http://localhost:8080/r4
-    module: onc_program
+  localhost_reference_server:
```

This preset was renamed to `localhost_reference_server` but it points to the reference server on `inferno.healthit.gov`.  Is this intentional? seems a little confusing...
```suggestion
  inferno_health_reference_server:
```

---

I named it this because the most important aspect of this entry is that it only shows up when running under `localhost`.  The second most important aspect is the server it is pointing to, but instead of having two host names in the key, I just state that it is an entry to the 'reference server'.

I'm not thrilled about this implementation (why do we need a key here?), but it works for now.